src/battle.ts

```diff
@@ -1085,6 +1085,9 @@ export class Battle {
 	endLastTurnPending = false;
 	totalTimeLeft = 0;
 	graceTimeLeft = 0;
+	currTurnTimeElapsed = 0;
+	timerMessageHighlighted = false;
```

Comment:
I don't think this needs two new properties on Battle? Like we're already displaying "User has X seconds left", we can just check how many seconds are left and use that to decide how pushy to make the message, right?